### PR TITLE
solves ianmurrays/ngDfp#51

### PR DESCRIPTION
**Pull request by LeonhardtDavid**
solves ianmurrays/ngDfp#51 (Google robot/spider fail)

### CODE REVIEW

**Comment by ianmurrays** [comment]
And the problem with spiders is just in that part of the script?

**Comment by LeonhardtDavid** [comment]
I used NoScript plugin for Firefox to block the scripts blocked by Google (took from the Google Webmaster Tool), before the change, the js breaks as the report said, after the change, the page loaded without problems (of course, with no banner, but the js didn't break).

File: angular-dfp.js [hunk 1]
```diff
@@ -263,14 +263,18 @@ angular.module('ngDfp', [])
...
-          }, self._refreshInterval());
+          if (self._refreshInterval() !== null) {
+            $interval(function () {
+              googletag.cmd.push(function() {
+                $window.googletag.pubads().refresh();
+              });
+            }, self._refreshInterval());
+          }
+        } catch (err) {
+          console.error(err);
```
Comment (ianmurrays) [inline angular-dfp.js]: I think this would be better as 

```javascript
} catch (err) {
  deferred.reject(err);
}
```

**Comment by LeonhardtDavid** [comment]
Added the change suggested.

**Comment by LeonhardtDavid** [comment]
The change with deferred.reject works fine.

**Comment by LeonhardtDavid** [comment]
@ianmurrays what do you think? It's ok or needs more modifications?

**Comment by ianmurrays** [comment]
Hi David, sorry about the delay, I've been too busy. I think for now it looks good. I still think the library needs a rewrite to account for the library not loading, but this will do for now 😄 

Gracias! Will merge and push the new version.

**Pull request merged**